diff --git a/node_modules/@eslint/eslintrc/dist/eslintrc-universal.cjs b/node_modules/@eslint/eslintrc/dist/eslintrc-universal.cjs
index 6000445..3761fbb 100644
--- a/node_modules/@eslint/eslintrc/dist/eslintrc-universal.cjs
+++ b/node_modules/@eslint/eslintrc/dist/eslintrc-universal.cjs
@@ -374,15 +374,11 @@ var ajvOrig = (additionalOptions = {}) => {
         meta: false,
         useDefaults: true,
         validateSchema: false,
-        missingRefs: "ignore",
         verbose: true,
-        schemaId: "auto",
         ...additionalOptions
     });
 
     ajv.addMetaSchema(metaSchema);
-    // eslint-disable-next-line no-underscore-dangle -- part of the API
-    ajv._opts.defaultMeta = metaSchema.id;
 
     return ajv;
 };
diff --git a/node_modules/@eslint/eslintrc/dist/eslintrc.cjs b/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
index 1b76091..8f1f1aa 100644
--- a/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
+++ b/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
@@ -1607,16 +1607,10 @@ var ajvOrig = (additionalOptions = {}) => {
         meta: false,
         useDefaults: true,
         validateSchema: false,
-        missingRefs: "ignore",
         verbose: true,
-        schemaId: "auto",
         ...additionalOptions
     });
 
-    ajv.addMetaSchema(metaSchema);
-    // eslint-disable-next-line no-underscore-dangle -- part of the API
-    ajv._opts.defaultMeta = metaSchema.id;
-
     return ajv;
 };
 
